packages/material-renderers/src/controls/MaterialInputControl.tsx

```diff
@@ -68,14 +69,20 @@ export const MaterialInputControl = (props: ControlProps & WithInput) => {
   const secondFormHelperText = showDescription && !isValid ? errors : null;
   const InnerComponent = input;
 
+  const InputMore: { label?: string } = {};
```

Comment:
```suggestion
  const additionalInputProps: WithInputProps = {};
```
It's very minor but could we not capitalize this variable because it is not a component and give it a more suitable name?
As I understand it this should contain the additional props defined in `WithInputProps`. Thus, I think it makes sense to reuse the type here to keep it consistent.

Reply:
Can be removed completely.